iDiff/cmd/root.go

```diff
@@ -19,16 +19,50 @@ var json bool
 var eng bool
 
 var RootCmd = &cobra.Command{
-	Use:   "[differ] [container1] [container2]",
+	Use:   "[differ] [image1] [image2]",
 	Short: "Compare two images.",
 	Long:  `Compares two images using the specifed differ (see iDiff documentation for available differs).`,
 	Run: func(cmd *cobra.Command, args []string) {
-		if validArgs, err := validateArgs(args[1:]); !validArgs {
+		if validArgs, err := validateArgs(args); !validArgs {
 			glog.Error(err.Error())
 			os.Exit(1)
 		}
-		if diff, err := differs.Diff(args[2], args[3], args[1], json, eng); err == nil {
-			fmt.Println(diff)
+		utils.SetDockerEngine(eng)
```

Comment:
Eventually I think this should be determined automatically by the tool (and not rely on a flag being passed in by the user). This is fine for now though.